### PR TITLE
docs: Supported/Recommended OS Documentation

### PR DESCRIPTION
**Pull request by taicchoumsft**
## Description
OS version matrix

## Task Item
#minor

### CODE REVIEW

**Review by cwhitten — DISMISSED** [review]

File: docs/1.3-supported-os.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
+# Composer 1.3 - Recommended OS versions
+
+The tables below provide recommended OS information for Composer 1.3.
+
+For this release, Composer will **only run on x64**. We plan to widen support to other architectures in upcoming releases.
```
Comment (zxyanliu) [inline docs/1.3-supported-os.md:5]: Suggestion:
We plan to support more architectures in upcoming releases.

File: docs/1.3-supported-os.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+## Linux
+
+OS                | Version            | Architectures |
+------------------|--------------------|---------------|
+Ubuntu            | 18.04, 20.04       | x64           |
+Debian            | 9+                 | x64           |
+
+
+## Supported NodeJS versions
```
Comment (zxyanliu) [inline docs/1.3-supported-os.md:32]: Node.js

File: docs/1.3-supported-os.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+If you are building from source, the tables below indicate the versions of NodeJS that Composer will run on.
```
Comment (zxyanliu) [inline docs/1.3-supported-os.md:34]: If you are building Composer from source, the table below shows the Node.js versions that Composer will run on.

File: docs/1.3-supported-os.md [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+
+NodeJS  |         Version            | Architectures |
```
Comment (zxyanliu) [inline docs/1.3-supported-os.md:36]: Node.js

**Comment by zxyanliu** [comment]
LGTM, thanks!

File: docs/1.3-supported-os.md [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+For this release, Composer will **only run on x64**. We will support more architectures in upcoming releases.
+
+## Windows
+
+OS                | Version            | Architectures |
+------------------|--------------------|---------------|
+Windows 10        | Version 1607+      | x64           |
+Windows Server    | 2012 R2+           | x64           |
+Nano Server       | Version 1803+      | x64           |
+Windows Client    | 7 (SP1), 8.1       | x64           |
```
Comment (boydc2014) [inline docs/1.3-supported-os.md:14]: I'm not sure we should recommend windows 7 or 8.1

File: docs/1.3-supported-os.md [hunk 6]
```diff
@@ -0,0 +1,40 @@
...
+
+
+## Supported Node.js versions
+
+If you are building Composer from source, the table below shows the Node.js versions that Composer will run on.
+
+Node.js |         Version            | Architectures |
+--------|----------------------------|---------------|
+LTS     | v10, v12, v14              | x64           |
+Non-LTS | v13                        | x64           |
```
Comment (boydc2014) [inline docs/1.3-supported-os.md:39]: Should we avoid claiming composer will work in non-LTS node.

**Comment by boydc2014** [comment]
Thanks @taicchoumsft for putting, looks good, a few comments listed above, generally i suggest we claim a little bit less and require a little bit newer OS version. I think requiring a newer version of OS won't be an issue with 99.99 users and will make our life easier. What do you guys think @cwhitten @zxyanliu

File: docs/1.3-supported-os.md (hunk 5 above)
Comment (taicchoumsft) [reply docs/1.3-supported-os.md:14]: sounds good, will take them out.

File: docs/1.3-supported-os.md (hunk 6 above)
Comment (taicchoumsft) [reply docs/1.3-supported-os.md:39]: I'll take it out for now.

**Review by pavolum — APPROVED** [review]

**Review by zxyanliu — APPROVED** [review]

**Pull request merged**